### PR TITLE
Latest files from Crowdin

### PR DESCRIPTION
**Pull request by zapbot**
Autogenerated PR from zapbot

### CODE REVIEW

File: src/org/zaproxy/zap/extension/pscanrulesBeta/resources/help_pt_BR/contents/pscanbeta.html [hunk 1]
```diff
@@ -7,73 +7,66 @@
...
+<p>Esta verificação identifica respostas onde o Content-Type do cabeçalho HTTP declara um charset diferente do charset definido pelo corpo do HTML ou XML. Quando há essa incompatibilidade de charsets do cabeçalho HTTP e do corpo os navegadores da Web podem ser forçados a entrar num indesejável sniffing de conteúdo para determinar o conjunto de caracteres correto daquele conteúdo.</p>
+<p>O scanner lida com várias condições dependendo do limite definido para o scanner, como segue:</p>
 <ul>
-  <li>All Thresholds:
+  <li>Todos os limiares:
     <ul>
-      <li>Header Versus Meta Content-Type Charset - The declaration in the HTTP Content-Type header doesn't match what is declared in a META Content-Type tag.</li>
-      <li>Header Versus Meta Charset - The declaration in the HTTP Content-Type header doesn't match what is declared in a META Charset tag.</li>
+      <li>Header Versus Meta Content-Type Charset - a declaração no cabeçalho HTTP Content-Type não combina com o que é declarado em uma tag META Content-Type.</li>
+      <li>Header Versus Meta Content-Type Charset - a declaração no cabeçalho HTTP Content-Type não combina com o que é declarado em uma tag META Charset.</li>
```
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/pscanrulesBeta/resources/help_pt_BR/contents/pscanbeta.html]: This change is incorrect, not sure how to address.

Comment (thc202) [inline src/org/zaproxy/zap/extension/pscanrulesBeta/resources/help_pt_BR/contents/pscanbeta.html]: Fixed in Crowdin.

**Comment by kingthorin** [comment]
So just close this one?

**Comment by thc202** [comment]
Right, @zapbot should create a new one with that fixed.

**Pull request closed without merging**